app/helpers/form_helper.py

```diff
@@ -12,7 +12,7 @@
 logger = get_logger()
 
 
-def get_form_for_location(block_json, location, answer_store, error_messages, disable_mandatory=False):
+def get_form_for_location(schema, block_json, location, answer_store, disable_mandatory=False):
```

Comment:
@jonnyshaw89 dont we need the messages here?

Reply:
no, we grab it from the schema later on.